static/css/external-site.less

```diff
@@ -0,0 +1,46 @@
+/* ==========================================================================
+   cfgov-refresh
+   Third-Party links pages at /external-site/.
+   ========================================================================== */
+
+.external-site_container {
+    .content_bar {
+      border-top: 1px solid @gray-50;
```

Comment:
thanks @jimmynotjim , fixed in https://github.com/cfpb/cfgov-refresh/commit/caf352add969a452fb3619be9405bb1df6cb2a0f